packages/components/src/composite/stories/index.story.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { isRTL } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Composite, useCompositeStore } from '..';
+import { UseCompositeStorePlaceholder, transform } from './utils';
+
+const meta: Meta< typeof UseCompositeStorePlaceholder > = {
+	title: 'Components/Composite',
+	component: UseCompositeStorePlaceholder,
+	subcomponents: {
+		// @ts-expect-error Storybook doesn't like overloaded exports as subcomponents
+		Composite,
+		// @ts-expect-error Storybook doesn't like overloaded exports as subcomponents
+		'Composite.Group': Composite.Group,
+		// @ts-expect-error Storybook doesn't like overloaded exports as subcomponents
+		'Composite.GroupLabel': Composite.GroupLabel,
+		// @ts-expect-error Storybook doesn't like overloaded exports as subcomponents
+		'Composite.Row': Composite.Row,
+		// @ts-expect-error Storybook doesn't like overloaded exports as subcomponents
+		'Composite.Item': Composite.Item,
```

Comment:
FWIW I think this is just an issue with the `subcomponents` typing and not overloaded exports specifically.

https://github.com/WordPress/gutenberg/blob/6581dfb8f1c605119db776f023a83d655209bfa8/packages/components/src/dropdown-menu-v2/stories/index.story.tsx#L38-L53

Reply:
That's weird. I remember seeing a different TS error last week 🤔 

Anyway, I checked again and I now get the same error as in `DropdownMenuV2` indeed — that's good! I updated the `// @ts-expect-error` comment to match the ones used for other storybook files.

https://github.com/WordPress/gutenberg/pull/63564/commits/922426e4461eee080ef46eec5690ffa3272330c3